src/panel/machine.ts

```diff
@@ -9,6 +9,8 @@ export type MachineEvents =
   | { type: 'FINISH'; results: TaskGroupResult[] }
   | { type: 'PIN' }
   | { type: 'UNPIN' }
+  | { type: 'SAVE' }
```

Comment:
Can we rename this event to `SAVE_TO_FILE` so that is nicely paired with `LOAD_FROM_FILE` ? :D

Reply:
Yep